### ISSUE TITLE
WebSocketError(<WSCloseCode.MESSAGE_TOO_BIG: 1009>

### ISSUE THREAD

**Issue by PoVPoV29** [hassio-addons/addon-node-red#1296]
# Problem/Motivation

> No node in node-red (no auto completion)

## Expected behavior

> find my entities and services in nod-red

## Actual behavior

> Node-red is empty

## Steps to reproduce

> it's a fresh install

## Proposed changes

I found some error repeat and repeat again in the add-on logs : 
`16 Feb 09:52:46 - [info] [server:Home Assistant] Connecting to http://supervisor/core
16 Feb 09:52:46 - [info] [server:Home Assistant] Connected to http://supervisor/core
16 Feb 09:52:47 - [info] [server:Home Assistant] Connection closed to http://supervisor/core
16 Feb 09:52:52 - [info] [server:Home Assistant] Connecting to http://supervisor/core
16 Feb 09:52:52 - [info] [server:Home Assistant] Connected to http://supervisor/core
16 Feb 09:52:53 - [info] [server:Home Assistant] Connection closed to http://supervisor/core`

And in the Core supervisor : 
`22-02-16 09:48:38 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize
22-02-16 09:48:38 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered
22-02-16 09:48:38 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running
22-02-16 09:48:40 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Received message 258:WebSocketError(<WSCloseCode.MESSAGE_TOO_BIG: 1009>, 'Message size 4911123 exceeds limit 4194304') is not str
22-02-16 09:48:40 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed`

Maybe it's because I have too many entities but I don't want to delete them.
Do you have any idea how to fix this ?
Thanks for your help

**Comment by djk314** [comment]
Mine too with update to 11.0.3 to 11.0.2, 11.0.2 working i have a backup. When i update it crash loop with a lot of errors. 

Welcome to Node-RED
===================
19 Feb 11:35:24 - [info] Node-RED version: v2.2.1
19 Feb 11:35:24 - [info] Node.js  version: v16.14.0
19 Feb 11:35:24 - [info] Linux 5.10.98 arm64 LE
19 Feb 11:35:25 - [info] Loading palette nodes
19 Feb 11:35:32 - [info] Dashboard version 3.1.6 started at /endpoint/ui
19 Feb 11:35:34 - [warn] ------------------------------------------------------
19 Feb 11:35:34 - [warn] [node-red-node-rbe/rbe] 'rbe' already registered by module node-red
19 Feb 11:35:34 - [warn] ------------------------------------------------------
19 Feb 11:35:34 - [info] Settings file  : /etc/node-red/config.js
19 Feb 11:35:34 - [info] Context store  : 'default' [module=memory]
19 Feb 11:35:34 - [info] User directory : /config/node-red/
19 Feb 11:35:34 - [warn] Projects disabled : editorTheme.projects.enabled=false
19 Feb 11:35:34 - [info] Flows file     : /config/node-red/flows.json
19 Feb 11:35:34 - [info] Server now running at http://127.0.0.1:46836/
19 Feb 11:35:34 - [info] Starting flows
19 Feb 11:35:37 - [info] Started flows
19 Feb 11:35:37 - [info] [server:Home Assistant] Connecting to http://supervisor/core
19 Feb 11:35:37 - [info] [mqtt-broker:Mqtt Broquer] Connected to broker: mqtt://192.168.0.30:1883
19 Feb 11:35:37 - [info] [alexa-remote-account:Alexa] intialising "Alexa" with the PROXY method and saved data...
19 Feb 11:35:37 - [info] [server:Home Assistant] Connected to http://supervisor/core
19 Feb 11:35:38 - [red] Uncaught Exception:
19 Feb 11:35:38 - [error] Error: Invalid context key
    at Object.value (/opt/node_modules/@node-red/runtime/lib/nodes/context/index.js:291:31)
    at EventsState.getContextValue (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/BaseNode.js:143:34)
    at EventsState.getComparatorResult (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/BaseNode.js:167:27)
    at EventsState.onHaEventsStateChanged (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/events-state/controller.js:71:32)
    at EventEmitter.emit (node:events:532:35)
    at Websocket.emitEvent (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/homeAssistant/Websocket.js:35:30)
    at Websocket.onClientEvents (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/homeAssistant/Websocket.js:254:18)
    at Object.callback (/config/node-red/node_modules/node-red-contrib-home-assistant-websocket/dist/homeAssistant/Websocket.js:188:65)
    at WebSocket.Connection._handleMessage (/config/node-red/node_modules/home-assistant-js-websocket/dist/haws.cjs:174:30)
    at WebSocket.onMessage (/config/node-red/node_modules/ws/lib/event-target.js:132:16)
    at WebSocket.emit (node:events:520:28)
    at Receiver.receiverOnMessage (/config/node-red/node_modules/ws/lib/websocket.js:1008:20)
    at Receiver.emit (node:events:520:28)
    at Receiver.dataMessage (/config/node-red/node_modules/ws/lib/receiver.js:517:14)
    at /config/node-red/node_modules/ws/lib/receiver.js:468:23
    at /config/node-red/node_modules/ws/lib/permessage-deflate.js:308:9
[11:35:39] INFO: Starting Node-RED...
> start
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"
19 Feb 11:35:41 - [info] 
Welcome to Node-RED
===================
19 Feb 11:35:41 - [info] Node-RED version: v2.2.1
19 Feb 11:35:41 - [info] Node.js  version: v16.14.0
19 Feb 11:35:41 - [info] Linux 5.10.98 arm64 LE
19 Feb 11:35:43 - [info] Loading palette nodes

**Comment by djk314** [comment]
I solved my problem with old backup. And i made an update in a lot of ways. The working one was : 
On home assistant 22.8 (with 22.9 pending)
- Updated home assistant plugins for nodered to 42.7 on nodered 11.02.
- Updated nodered to 11.03
- Updated home assistant to 22.9
All work but i've a lot of debuging things in nodered log console now, not errors but a lot of infos, like an hight debug level.
I always have node_red_node_rbe filter 0.5.0 plugins that say "type already registred" in nodered but i saw it was merged with filter, the problem is to remove it, each time i delete the plug and in some config files even when nodered stopped the error is here again, but all working.
